### PR TITLE
Smartcam capture function

### PR DESCRIPTION
**Pull request by turbokongen**
WIP, Adding capture functionallity from mypages. Capture is now working.

usage:
import verisure
mypages = verisure.MyPages('username', 'password')
mypages.login()

mypages.smartcam.capture('device_serial_number') -> captures a new image to mypages.
mypages.smartcam.get_image_list() -> get a list of image names to use for downloading.
mypages.smartcam.download_image('device_id', 'image_name', destination_path) -> Download the specifyed image to destination_path.

Any other things that might be good?

### CODE REVIEW

File: verisure/devices/smartcam.py [hunk 1]
```diff
@@ -22,3 +26,43 @@ def get(self):
...
+        for key in status:
+            if key == 'totalAmount':
+                total_images = status['totalAmount']
+                print('Total amount of images available for download:',
+                      total_images)
+        image_series = status['imageSeries']
+        image_data_list = [li['images'] for li in image_series]
+        n = len(image_data_list)
+        image_ids = []
+        for i in range(0, n):
```
Comment (persandstrom) [inline verisure/devices/smartcam.py]: can this be written as ```image_ids = [image_data['id'] for image_data in image_data_list] ```

Comment (turbokongen) [inline verisure/devices/smartcam.py]: Doing it that way make it fail with `TypeError: list indicies must be integers, not str`

Comment (persandstrom) [inline verisure/devices/smartcam.py]: Aha. Keep it like it is then

File: verisure/devices/smartcam.py [hunk 2]
```diff
@@ -22,3 +26,43 @@ def get(self):
         """ Get device overview """
         status = self._session.get(OVERVIEW_URL)
         return [Overview('smartcam', val) for val in status]
+
+    def download_image(self, device_id, image_id):
+        """Download a image from mypages smartcam."""
+        pic_url = (DOMAIN + DOWNLOAD_URL.format(
+            device_id.upper().replace(' ', '%20'),
+            image_id))
+        image = self._session._session.get(pic_url, stream=True)
```
Comment (turbokongen) [inline verisure/devices/smartcam.py]: Added :+1:

File: verisure/devices/smartcam.py [hunk 3]
```diff
@@ -22,3 +24,34 @@ def get(self):
...
         status = self._session.get(OVERVIEW_URL)
         return [Overview('smartcam', val) for val in status]
+
+    def download_image(self, device_id, image_id, dest_path):
+        """Download a image from mypages smartcam."""
+        pic_url = DOWNLOAD_URL.format(
+            device_id.upper().replace(' ', '%20'),
+            image_id)
+        image_filename = pic_url.rsplit('/', 1)[1]
+        self._session.download(pic_url, (dest_path + image_filename))
```
Comment (persandstrom) [inline verisure/devices/smartcam.py]: Use ```os.path.join``` here, then the user can write ```/tmp``` or ```/tmp/``` as dest_path

**Comment by persandstrom** [comment]
Looks good! I will make a quick release of this and update HA dev at once

**Pull request merged**